src/server/bundle.ts

```diff
@@ -11,7 +11,7 @@ export interface JSXConfig {
 let esbuildInitialized: boolean | Promise<void> = false;
 async function ensureEsbuildInitialized() {
   if (esbuildInitialized === false) {
-    if (Deno.run === undefined) {
+    if (Deno.Command === undefined) {
```

Comment:
I would not like to change this specific check right now, due to some future proofing work I need to do in `x/esbuild_deno_loader`. Can you revert this specific change?